### PR TITLE
feat: update styling in admin page

### PR DESCRIPTION
**Pull request by InbarDanieli**

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import type { AppProps } from "next/app";
+import "../styles/globals.css";
```
Comment (OmriBarZik) [inline pages/_app.tsx:2]: please move this import to the top of the file

File: pages/admins/index.style.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { css } from "@emotion/react";
```
Comment (OmriBarZik) [inline pages/admins/index.style.ts:1]: please use `import {css} from '@mui/material'`

**Review by OmriBarZik — CHANGES_REQUESTED** [review]

File: pages/admins/index.style.ts (hunk 2 above)
Comment (InbarDanieli) [reply pages/admins/index.style.ts]: done

File: pages/_app.tsx (hunk 1 above)
Comment (InbarDanieli) [reply pages/_app.tsx:1]: done

**Review by OmriBarZik — APPROVED** [review]

**Pull request merged**